sqlitecluster/SQLiteNode.cpp

```diff
@@ -1371,22 +1559,53 @@ bool SQLiteNode::update(uint64_t& nextActivity) {
                             SASSERTWARN(_currentCommand->transaction.empty());
                             SINFO("Starting processing command '" << _currentCommand->request.methodLine << "' ("
                                                                   << _currentCommand->id << ")");
-                            _processCommandWrapper(_db, _currentCommand);
+
+                            bool unlock = false;
```

Comment:
(Looking closer at this, it looks like it an auto-unlocking pattern might not work here...)